content/docs/1.11.0/nodes-and-volumes/volumes/rwx-volumes.md

```diff
@@ -7,30 +7,35 @@ Longhorn supports ReadWriteMany (RWX) volumes by exposing regular Longhorn volum
 
 ## Introduction
 
-Longhorn provides two types of RWX volumes, each optimized for different workload requirements.
+Longhorn provides two types of RWX volumes, each optimized for different workload requirements:
 
-- **Generic (Non-Migratable) RWX Volumes**
+- **Generic (Non-Migratable) RWX Volumes** - traditional NFS-based shared filesystem volumes.
+- **Migratable RWX Volumes** - specialized RWX volumes designed for KubeVirt VM live migration.
 
-  Generic RWX volumes provide traditional shared filesystem access across multiple nodes. They use dedicated NFSv4.1 servers running in `share-manager-<volume-name>` Pods within the longhorn-system namespace. Each RWX volume is paired with a corresponding Service that exposes the NFS endpoint to clients.
+Migratable RWX volumes are often misunderstood or overlooked in deployments involving virtualization platforms such as KubeVirt. Users sometimes incorrectly set `spec.migratable: true` on volumes that are not intended for migration, or remain unaware that Longhorn supports KubeVirt live migration at all.  
```

Comment:
Redundant information. We can remove it because we have explained the two types in the following content.